x-pack/plugins/security_solution/public/resolver/view/process_event_dot.tsx

```diff
@@ -410,39 +438,31 @@ const ProcessEventDotComponents = React.memo(
         type: 'userSelectedResolverNode',
         payload: {
           nodeId,
+          selectedProcessId: selfId,
         },
       });
-    }, [animationTarget, dispatch, nodeId]);
+      pushToQueryParams({ crumbId: selfEntityId, crumbEvent: 'all' });
+    }, [animationTarget, dispatch, nodeId, selfEntityId, pushToQueryParams, selfId]);
 
-    const handleRelatedEventRequest = useCallback(() => {
-      dispatch({
-        type: 'userRequestedRelatedEventData',
-        payload: event,
-      });
-    }, [dispatch, event]);
-
-    const handleRelatedAlertsRequest = useCallback(() => {
-      dispatch({
-        type: 'userSelectedRelatedAlerts',
-        payload: event,
-      });
-    }, [dispatch, event]);
     /**
      * Enumerates the stats for related events to display with the node as options,
      * generally in the form `number of related events in category` `category title`
      * e.g. "10 DNS", "230 File"
      */
-    const relatedEventOptions = useMemo(() => {
+
+    const [relatedEventOptions, grandTotal] = useMemo(() => {
       const relatedStatsList = [];
 
       if (!relatedEventsStats) {
         // Return an empty set of options if there are no stats to report
-        return [];
+        return [[], 0];
       }
+      let runningTotal = 0;
```

Comment:
Similar to my other comment about calculating totals in the view: this seems like biz logic and we should:
1. separate it from the view logic here
2. centralize it in a model or in selectors (where we'll know to look for it, and where it can be easily tested w/o mocking and w/o running react code.)